### DISCUSSION TITLE
Have there been any bans while running the code from a terminal?

### DISCUSSION THREAD

**Discussion by TriOCuBe** [Kyrluckechuck/TFT-Bot#216] [Q&A]
Title. Ive read the warning and the past discussions about bans, and i know the .exe was pretty suspectible to them, but have there been any while only running the code itself? Im guessing not, since i dont see how the game should figure out youre botting, but im asking anyways.

**Comment by Kyrluckechuck** [comment]
@TriOCuBe I have not seen any recent reports about this since we migrated away from using the compiled release/`.exe` approach, and while I can't say you won't get banned by using it since it's directly violating their ToS, it _seems_ to be ignored!
